libs/frontend/feature-build-event/src/lib/use-build-event.ts

```diff
@@ -220,7 +221,8 @@ export const useBuildEvent = () => {
       // This is for elements that use an additional wrapper for hover edges, such as img.
       // Kebab case "component-id" will be converted to camel case "componentId".
       // See https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#name_conversion
-      resolveComponent(site.value.context, target?.dataset.componentId)
+      resolveComponent(site.value.context, target?.dataset.componentId) ??
+      editor?.resizeData?.component
```

Comment:
This is to fix a render issue when the cursor is not hovering on the image during resize.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/a2e922c2-76f3-4bd0-9083-f840c0173912